manim/mobject/geometry/line.py

```diff
@@ -85,16 +88,19 @@ def set_points_by_ends(
         """
         self._set_start_and_end_attrs(start, end)
         if path_arc:
+            # self.path_arc could potentially be None, which is not accepted
+            # as parameter.
+            assert self.path_arc is not None
             arc = ArcBetweenPoints(self.start, self.end, angle=self.path_arc)
             self.set_points(arc.points)
         else:
-            self.set_points_as_corners([self.start, self.end])
+            self.set_points_as_corners(np.array([self.start, self.end]))
```

Comment:
Is the copy necessary? `np.asarray` might be a better way to go if not.

Reply:
Implemented.